### PR TITLE
Add 4-channel (RGBD) evaluation

### LINKED ISSUE #28: Trained model for testing purpose
Hi Author,

Any Chance you guys are releasing trained-model as well.

Or the CASIA-SURF dataset itself. ?

### PR DESCRIPTION
**Pull request by trushkin-ilya**

### CODE REVIEW

**Comment by grib0ed0v** [comment]
@trushkin-ilya permission for GDrive should be changed to download without requesting access each time for each new user.

**Comment by trushkin-ilya** [comment]
@grib0ed0v, I changed permissions. Could you please check that you can access links.

File: test.py [hunk 1]
```diff
@@ -58,25 +59,10 @@ def main(args):
     model = getattr(models, args.model)(num_classes=args.num_classes)
```
Comment (grib0ed0v) [inline test.py]: num_classes=config['num_classes'] ?

File: models/__init__.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
 from .ensemble import Ensemble
 from torchvision.models import *
```
Comment (grib0ed0v) [inline models/__init__.py]: Why do we need here to import * from torchvision.models?

File: find_lr.py [hunk 3]
```diff
@@ -1,56 +1,74 @@
...
 
 
 if __name__ == '__main__':
     argparser = ArgumentParser()
-    argparser.add_argument('--model', required=True, type=str)
-    argparser.add_argument('--num_classes', type=int, default=2)
-    argparser.add_argument('--depth', type=bool, default=False)
-    argparser.add_argument('--ir', type=bool, default=False)
+    argparser.add_argument('--config_path', required=True, type=str)
+    argparser.add_argument('--optimizer', default="Adam", type=str)
```
Comment (grib0ed0v) [inline find_lr.py]: Looks like we have a duplication here - optimizer is already written in config file.

File: find_lr.py [hunk 4]
```diff
@@ -1,56 +1,74 @@
...
+    argparser.add_argument('--loss_fn', default="CrossEntropyLoss", type=str)
```
Comment (grib0ed0v) [inline find_lr.py]: The same as above.

**Review by grib0ed0v — CHANGES_REQUESTED** [review]
Need to fix existing comments.

File: models/__init__.py (hunk 2 above)
Comment (trushkin-ilya) [reply models/__init__.py]: That allows any model class defined in `torchvision.models` to use in config files.

**Review by grib0ed0v — APPROVED** [review]

**Pull request merged**